readme.md

```diff
@@ -19,7 +18,19 @@
 
 ## Inspect the Dashboard Performance in the Dashboard Designer
 
+1. Download the [Dashboard Diagnostic Tool](https://github.com/DevExpress-Examples/bi-dashboard-diagnosic-tool). 
+
+2. Reference `DiagnosticTool.dll` and install the [Microsoft.Diagnostics.Tracing.TraceEvent](https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.TraceEvent/) package in your dashboard project. 
+
+3. Create a custom button and insert it to the Ribbon.
+
+4. Create a `DiagnosticController` object. 
+
+5. Call the conroller's Start() and Stop() methods on button's click to run and finish the Dashboard Diagnostic Tool's session.
+
+6. Implement the `IFileController` interface and specify the output file path in the `TrySaveFile` method. Pass a new class instance that implements `IFileController` to the controller's contructor. 

```

Comment:
1. "contructor" is misspelled. Please change it to "constructor".

2. I don't think "implements `IFileController` _to_" is correct.
Do you mean "implements `IFileController` _in_ the controller's constructor"?

Reply:
```suggestion
6. Implement the `IFileController` interface and specify the output file path in the `TrySaveFile` method. Pass a new class instance that implements `IFileController` in the controller's constructor. 

```